pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -309,6 +309,23 @@ type SeedSpec struct {
 	//nolint:staticcheck
 	//lint:ignore SA5008 omitcegenyaml is used by the example-yaml-generator
 	KubeLB *KubeLBSettings `json:"kubelb,omitempty,omitcegenyaml"`
+	// MetrcisCollectors toggles metrics collectors of the seed controller manager.
+	MetrcisCollectors *MetrcisCollectors `json:"metrcisCollectors,omitempty"`
+}
+
+type MetrcisCollectors struct {
+	// Addon enables addon metrics collector
+	// If not set, then the value from global config is used.
+	Addon *bool `json:"addon,omitempty"`
+	// ClusterBackup enables cluster backup metrics collector
+	// If not set, then the value from global config is used.
+	ClusterBackup *bool `json:"clusterBackup,omitempty"`
+	// Cluster enables cluster metrics collector
+	// If not set, then the value from global config is used.
+	Cluster *bool `json:"cluster,omitempty"`
+	// Project enables project metrics collector
+	// If not set, then the value from global config is used.
+	Project *bool `json:"project,omitempty"`
```

Comment:
And in the Seed we have a completely different structure for enabling the collectors? Why is this struct a different thing than in the KubermaticConfiguration?

Reply:
changed